podman/tests/utils.py

```diff
@@ -0,0 +1,13 @@
+import re
+import pathlib
+import csv
+
+
+def get_os_release() -> dict:
+    path = pathlib.Path("/etc/os-release")
```

Comment:
Why didn't you use the [`platform`](https://docs.python.org/3/library/platform.html#module-platform) library?

Reply:
`platform.freedesktop_os_release()`

Reply:
thanks @Honny1, I changed the tests using `platform`